React/Base/RCTConvert.m

```diff
@@ -644,15 +655,9 @@ + (UIFont *)UIFont:(UIFont *)font withFamily:(id)family
       isItalic = style ? isItalic : RCTFontIsItalic(font);
       isCondensed = RCTFontIsCondensed(font);
     } else {
-      // Not a valid font or family
+      // Not a valid font or family, fallback with system one
       RCTLogError(@"Unrecognized font family '%@'", familyName);
-      if ([UIFont respondsToSelector:@selector(systemFontOfSize:weight:)]) {
-        font = [UIFont systemFontOfSize:fontSize weight:fontWeight];
-      } else if (fontWeight > UIFontWeightRegular) {
-        font = [UIFont boldSystemFontOfSize:fontSize];
-      } else {
-        font = [UIFont systemFontOfSize:fontSize];
-      }
```

Comment:
My commit msg is wrong actually - I assumed `systemFontOfSize` was iOS8+ which in fact only applies to `systemFontOfSize:weight`. However I still think it might be easier to just call the method again without fontFamily.